plugins/Kaleidoscope-LEDControl/README.md

```diff
@@ -1,6 +1,118 @@
 # Kaleidoscope-LEDControl
 
 This is a plugin for [Kaleidoscope][fw], for controlling the LEDs, and LED
-effects.
+effects. It is also a building block for plugins that control LEDs.
 
  [fw]: https://github.com/keyboardio/Kaleidoscope
+
+## Using the extension
+
+## Plugin methods
+
+### `.next_mode(void)`
+
+> Activates the next LED mode. Cycles to the first LED mode if the current LED
+> mode is the last one.
+
+### `.prev_mode(void)`
+
+> Activates the previous LED mode. Cycles to the last LED mode if the current
+> LED mode is the first one.
+
+### `.get_mode()`
```

Comment:
There are two overloads for this method and I'm yet to understand what one of them does. Asked the question on Discord, but leaving this here as a reminder.